lighthouse-core/config/default.js

```diff
@@ -7,8 +7,24 @@
 
 /* eslint-disable max-len */
 
+const emulation = require('../lib/emulation');
+
 module.exports = {
-  settings: {},
+  settings: {
+    onlyAudits: ['first-contentful-paint'],
+    throttling: {
+      // method: 'devtools',
+      // requestLatency: emulation.settings.TYPICAL_MOBILE_THROTTLING_METRICS.latency,
```

Comment:
I named them differently specifically because they operate at different levels and the amount you'd want to apply to one *should* be very different from the amount you'd apply to the other to get the same experience :)